lib/index.js

```diff
@@ -21,6 +21,7 @@ module.exports = class Mdify {
      *
      *  - `start` (String): The start delimiter of the metadata (default: `---`).
      *  - `end` (String): The end delimiter of the metadata (default: `---`).
+     *  - `yamlOptions` (Object): Custom ys-yaml options.
```

Comment:
Just a small typo, will fix it. `ys` :arrow_right: `js`

Reply:
Good catch! :)